FordLabs/peoplemover
# Move isUnassignedDrawerOpen state from redux to recoil

## Issue
Resolves N/A

## What was done
- [x] Move isUnassignedDrawerOpen state from redux to recoil

## How to test
1. Ensure that if the unassigned drawer is closed and you add a new person, the drawer opens when you save the new person.
